great_expectations/dataset/base.py

```diff
@@ -97,14 +98,23 @@ def wrapper(self, *args, **kwargs):
                         del all_args["result_format"]
 
                 all_args = recursively_convert_to_json_serializable(all_args)
-                expectation_args = copy.deepcopy(all_args)
+
+                # Patch in PARAMETER args, and remove locally-supplied arguments
+                expectation_args = copy.deepcopy(all_args) # This will become the stored config
+
+                if "meta" in self._expectations_config and "evaluation_parameters" in self._expectations_config["meta"]:
+                    evaluation_args = self._build_evaluation_parameters(expectation_args,
+                                                                        self._expectations_config["meta"]["evaluation_parameters"]) # This will be passed to the evaluation
```

Comment:
Oh! I didn't realize eval_params wasn't its own top-level object. What does putting it in meta buy us?

Reply:
haha...I'm working through the comments, and had just wondered the same thing in a previous comment. Sounds like this answers it...I'll move it to a top-level key. I think that's clearer. (I hadn't put it there originally, because of the way we were looking at there being per-expectation keys in the meta object from an older design for this feature).

Reply:
:+1: